tests/fixtures

```diff
@@ -0,0 +1 @@
+../fixtures
```

Comment:
this is a (temporary) symlink -- some stuff in `getsentry` hardcodes `tests/fixtures` and I'll have to fix those in a 3-step process

Reply:
haha, classic